--- eggs/plone.app.querystring-1.0.7-py2.7.egg/plone/app/querystring/querybuilder.py.orig	2016-06-15 13:40:49.306174129 -0400
+++ eggs/plone.app.querystring-1.0.7-py2.7.egg/plone/app/querystring/querybuilder.py	2016-06-15 13:53:42.626814805 -0400
@@ -1,5 +1,7 @@
 import json
 
+from Products.ZCatalog.Catalog import CatalogError
+
 from zope.component import getMultiAdapter, getUtility
 from zope.i18n import translate
 
@@ -46,10 +48,15 @@
     def html_results(self, query):
         """html results, used for in the edit screen of a collection,
            used in the live update results"""
-        options = dict(original_context=self.context)
-        results = self(query, sort_on=self.request.get('sort_on', None),
+        try:
+            options = dict(original_context=self.context)
+            results = self(query, sort_on=self.request.get('sort_on', None),
                        sort_order=self.request.get('sort_order', None),
                        limit=10)
+        except KeyError:
+            raise KeyError('Nope')
+        except CatalogError:
+            raise CatalogError('Nope')
 
         return getMultiAdapter((results, self.request),
             name='display_query_results')(**options)
--- eggs/Products.CMFPlone-4.2.4-py2.7.egg/Products/CMFPlone/skins/plone_scripts/folder_copy.cpy.orig	2016-06-15 15:03:12.956116381 -0400
+++ eggs/Products.CMFPlone-4.2.4-py2.7.egg/Products/CMFPlone/skins/plone_scripts/folder_copy.cpy	2016-06-16 12:39:55.169887019 -0400
@@ -14,6 +14,13 @@
 from Products.CMFPlone import PloneMessageFactory as _
 
 REQUEST=context.REQUEST
+orig_template = REQUEST.get('orig_template', None)
+if orig_template and (not context.portal_url.isURLInPortal(orig_template) or 'iframe' in orig_template):
+    message = u'Nope'
+    context.plone_utils.addPortalMessage(message, 'error')
+    REQUEST.RESPONSE.redirect(context.portal_url())
+    return None
+    
 if 'paths' in REQUEST:
     ids = [p.split('/')[-1] or p.split('/')[-2] for p in REQUEST['paths']]
--- eggs/eke.publications-1.1.16-py2.7.egg/eke/publications/browser/rdf.py.orig    2016-11-08 14:33:45.000000000 -0600
+++ eggs/eke.publications-1.1.16-py2.7.egg/eke/publications/browser/rdf.py  2016-11-08 14:33:56.000000000 -0600
@@ -19,7 +19,7 @@
 _logger = logging.getLogger(__name__)
 
 # Constants
-FETCH_GROUP_SIZE = 450 # Fetch this many publications in Entrez.fetch, pausing to construct objects between each
+FETCH_GROUP_SIZE = 100 # Fetch this many publications in Entrez.fetch, pausing to construct objects between each
 
 # Other constnats: Well-known URI refs
 _publicationTypeURI = URIRef('http://edrn.nci.nih.gov/rdf/types.rdf#Publication')
