--- eggs/plone.app.querystring-1.0.7-py2.7.egg/plone/app/querystring/querybuilder.py.orig	2016-11-09 10:27:09.982602414 -0500
+++ eggs/plone.app.querystring-1.0.7-py2.7.egg/plone/app/querystring/querybuilder.py	2016-11-09 10:39:56.129966648 -0500
@@ -1,5 +1,8 @@
 import json
 
+from Products.ZCatalog.Catalog import CatalogError
+
+
 from zope.component import getMultiAdapter, getUtility
 from zope.i18n import translate
 
@@ -46,10 +49,15 @@
     def html_results(self, query):
         """html results, used for in the edit screen of a collection,
            used in the live update results"""
-        options = dict(original_context=self.context)
-        results = self(query, sort_on=self.request.get('sort_on', None),
+        try:
+            options = dict(original_context=self.context)
+            results = self(query, sort_on=self.request.get('sort_on', None),
                        sort_order=self.request.get('sort_order', None),
                        limit=10)
+        except KeyError:
+            raise KeyError('Nope')
+        except CatalogError:
+            raise CatalogError('Nope')
 
         return getMultiAdapter((results, self.request),
             name='display_query_results')(**options)
--- eggs/Products.CMFPlone-4.2.4-py2.7.egg/Products/CMFPlone/skins/plone_scripts/folder_copy.cpy.orig	2016-11-09 10:27:53.867032245 -0500
+++ eggs/Products.CMFPlone-4.2.4-py2.7.egg/Products/CMFPlone/skins/plone_scripts/folder_copy.cpy	2016-11-09 10:40:44.311289219 -0500
@@ -14,6 +14,13 @@
 from Products.CMFPlone import PloneMessageFactory as _
 
 REQUEST=context.REQUEST
+orig_template = REQUEST.get('orig_template', None)
+if orig_template and (not context.portal_url.isURLInPortal(orig_template) or 'iframe' in orig_template):
+    message = u'Nope'
+    context.plone_utils.addPortalMessage(message, 'error')
+    REQUEST.RESPONSE.redirect(context.portal_url())
+    return None
+    
 if 'paths' in REQUEST:
     ids = [p.split('/')[-1] or p.split('/')[-2] for p in REQUEST['paths']]
 
